extractor/build.gradle.kts

```diff
@@ -0,0 +1,143 @@
+/*
+ * SPDX-FileCopyrightText: 2025 NewPipe e.V. <https://newpipe-ev.de>
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    alias(libs.plugins.google.protobuf)
+    checkstyle
+    `maven-publish`
+}
+
+java {
+    withSourcesJar()
+    withJavadocJar()
+}
+
+// Prevent .proto files ending up in JARs
+tasks.jar {
+    exclude("**/*.proto")
+    includeEmptyDirs = false
+}
+
+tasks.test {
+    // Test logging setup
+    testLogging {
+        events = setOf(TestLogEvent.SKIPPED, TestLogEvent.FAILED)
+        showStandardStreams = true
+        exceptionFormat = TestExceptionFormat.FULL
+    }
```

Comment:
Why is this only in the extractor `build.gradle`? I think it was fine to have it for all subprojects (although yeah we don't have tests elsewhere)